Localization/Italian/ItalianInstaller.iss

```diff
@@ -0,0 +1,29 @@
+﻿[CustomMessages]
+; InstallType
+english.InstallType=Seleziona tipo installazione

```

Comment:
@bovirus thank you for localization!
The minor issue here is that strings in this file should start from `italian`, not `english`.